src/server/services/controller.test.ts

```diff
@@ -192,3 +204,57 @@ describe('controller', () => {
     expect(result).toEqual(data);
   });
 });
+
+describe('request handlers', () => {
```

Comment:
Nog een test voor wanneer profileType undefined/onbekend iets is?

Reply:
Nee want in de controller is ProfileType nooit onbekend. Zonder auth kun je het stream endpoint niet laden (401).

De functie in cms-content.ts kan echter wel met een arbitrair profileType geladen worden, vandaar de fallback.